apps/web/src/pages/Applications/ApplicationReviewPage/ReviewSection.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { Heading, Link } from "@gc-digital-talent/ui";
+import { useIntl } from "react-intl";
+
+interface ReviewSectionProps {
+  title: string;
+  path: string;
+  children: React.ReactNode;
+}
+
+const ReviewSection = ({ title, path, children }: ReviewSectionProps) => {
+  const intl = useIntl();
+  return (
+    <section data-h2-margin-bottom="base(x3)">
+      <div
+        data-h2-display="base(flex)"
+        data-h2-justify-content="base(space-between)"
+        data-h2-align-items="base(center)"
+      >
+        <Heading level="h3" data-h2-margin-top="base(0)">
+          {title}
+        </Heading>
+        <Link type="button" mode="inline" color="secondary" href={path}>
+          {intl.formatMessage({
+            defaultMessage: "Edit this section",
+            id: "mu6uvE",
+          })}
+        </Link>
```

Comment:
We will need to add more context (even if hidden) to this link. Having the [same text content with a different purpose](https://www.w3.org/WAI/standards-guidelines/act/rules/b20e66/proposed/) causes issues for AT users. Users have the ability to list all links on the page and when they hear "Edit this section" x4 it won't be clear what "this" is.

Reply:
Okay I can add an aria-label that has more context within each link. Added in [2e661b1](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6600/commits/2e661b123eba5652daa0bcf10b36e481d87dc678)